avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
+
```

Comment:
According the https://travis-ci.org/avocado-framework/avocado/jobs/178775845#L4201 the first commit has no effect on the issue, because you can see that the error occurred, but the first commit would not be in use.

I'll take another look at the second commit to see, whether it fixes the issue.